Configure

```diff
@@ -238,7 +238,8 @@ my $local_config_envname = 'OPENSSL_LOCAL_CONFIG_DIR';
 
 $config{sourcedir} = abs2rel($srcdir);
 $config{builddir} = abs2rel($blddir);
-$config{FIPSKEY} = '00';
+$config{FIPSKEY} =
+    'f4556650ac31d35461610bac4ed81b1a181b2d8a43ea2854cbae22ca74560813';
```

Comment:
May I suggest adding the adding the reference as a comment?
```suggestion
$config{FIPSKEY} =
#   SHA256("holy hand grenade of antioch")
    'f4556650ac31d35461610bac4ed81b1a181b2d8a43ea2854cbae22ca74560813';
```

Reply:
Updated commit pushed.